### PR TITLE
Update required python version to 3.7

### PR DESCRIPTION
**Pull request by drjohnnyfever1**
This is a naive bump of the required python version to 3.7 to match ports/head. 
This is working for me but there may be other things that need to be addressed. 

Perhaps it would be nice to support python3.6 and python3.7 for quarterly and master ports branches respectively?   

- [x] Explain the feature
- [x] Read [CONTRIBUTING.md](https://github.com/iocage/iocage/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,15 +15,15 @@ technologies the FreeBSD operating system has to offer. It is geared for ease
...
+To install using binary packages, simply run: `pkg install py37-iocage`
 
 ## Installation
 
 ### GitHub:
 
 The FreeBSD source tree ***must*** be located at `$SRC_BASE` (`/usr/src` by default) to build from git.
 
-- `pkg install python36 git-lite py36-cython py36-pip`
+- `pkg install python37 git-lite py37-cython py37-pip`
```
Comment (jsegaert) [inline README.md:26]: Maybe add package `py37-libzfs` here, instead of it being obscured in the Makefile

File: README.md [hunk 2]
```diff
@@ -15,15 +15,15 @@ technologies the FreeBSD operating system has to offer. It is geared for ease
...
 - `git clone --recursive https://github.com/iocage/iocage`
```
Comment (jsegaert) [inline README.md:27]: The `--recursive` option is no longer needed

File: doc/source/install.rst [hunk 3]
```diff
@@ -24,7 +24,7 @@ at :samp:`$SRC_BASE` ( :samp:`/usr/src` by default).
 
 To install from github, run these commands:
 
-:samp:`pkg install python36 git-lite libgit2 cython3 py36-pip`
+:samp:`pkg install python37 git-lite libgit2 cython3 py37-pip`
```
Comment (jsegaert) [inline doc/source/install.rst]: Some of these packages are no longer valid, `libgit2` is no longer needed and `cython3` has been renamed to `py37-cython`
This is a good opportunity to clean this up and make it consistent with the README.md      :smile:

File: iocage.docset/Contents/Resources/Documents/_sources/install.rst.txt [hunk 4]
```diff
@@ -24,7 +24,7 @@ at :samp:`$SRC_BASE` ( :samp:`/usr/src` by default).
 
 To install from github, run these commands:
 
-:samp:`pkg install python36 git-lite libgit2 cython3 py36-pip`
+:samp:`pkg install python37 git-lite libgit2 cython3 py37-pip`
```
Comment (jsegaert) [inline iocage.docset/Contents/Resources/Documents/_sources/install.rst.txt]: Same remark here for `libgit2` and `cython3`

File: iocage.docset/Contents/Resources/Documents/install.html [hunk 5]
```diff
@@ -58,7 +58,7 @@ <h2>Using github<a class="headerlink" href="#using-github" title="Permalink to t
 <p>If installing from github, the FreeBSD source tree <strong>must</strong> be located
 at <code class="samp docutils literal notranslate"><span class="pre">$SRC_BASE</span></code> ( <code class="samp docutils literal notranslate"><span class="pre">/usr/src</span></code> by default).</p>
 <p>To install from github, run these commands:</p>
-<p><code class="samp docutils literal notranslate"><span class="pre">pkg</span> <span class="pre">install</span> <span class="pre">python36</span> <span class="pre">git-lite</span> <span class="pre">libgit2</span> <span class="pre">cython3</span> <span class="pre">py36-pip</span></code></p>
+<p><code class="samp docutils literal notranslate"><span class="pre">pkg</span> <span class="pre">install</span> <span class="pre">python37</span> <span class="pre">git-lite</span> <span class="pre">libgit2</span> <span class="pre">cython3</span> <span class="pre">py36-pip</span></code></p>
```
Comment (jsegaert) [inline iocage.docset/Contents/Resources/Documents/install.html]: Same remark here for `libgit2` and `cython3` and there's an occurrence left to 36 in `py36-pip`

**Review by jsegaert — COMMENTED** [review]
Thank you so much for going through this exercise! 
I was working on a similar PR, but you beat me to it, so I took the liberty to review your changes and add my 2 cents.

File: doc/source/install.rst (hunk 3 above)
Comment (drjohnnyfever1) [reply doc/source/install.rst]: Roger that. I'll clean these up and update the PR

**Pull request closed without merging**